Userland/Libraries/LibJS/Tests/classes/class-methods.js

```diff
@@ -61,3 +61,66 @@ test("method named 'async'", () => {
     expect("async" in a).toBeTrue();
     expect(a.async()).toBe("function named async");
 });
+
+test("can call other private methods from methods", () => {
+    class A {
+        #a() {
+            return 1;
+        }
+
+        async #b() {
+            return 2;
+        }
+
+        normal_a() {
```

Comment:
Let's not use `snake_case` in JS code, I guess `normalA` is the closest approximation.